subprojects/programmatic-test/programmatic-test.gradle

```diff
@@ -16,3 +16,7 @@ tasks.javadoc.enabled = false
 
 sourceCompatibility = 11
 targetCompatibility = 11
+
+test {
```

Comment:
@TimvdLippe note on this change: I have not found the exact problem yet, but there are 2 test suites which manipulate mock makers. It seems like the order matters, I see the random test failures often. With clean JVM process (`forkEvery = 1`) not a single failure observed.

Reply:
Okay, we will monitor CI, but if you don't observe any failures locally, we are probably good to go here. Thanks for figuring that one out!